code/index.html

```diff
@@ -4,27 +4,88 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
+    <link rel="stylesheet" href="style.css">
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+      <header>
```

Comment:
This is where you would put the hero image. Instead of doing a img tag within the header tag you can use the background-image attribute. Using that wont make your image stretch like it does now when you expand the window.